src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -45,6 +45,22 @@
 logger = logging.getLogger(__name__)
 
 
+def _escape_version(version: str) -> str:
+    """Escape version as needed to cover edge cases."""
```

Comment:
The docstring should be clear about what the function does. "as needed to cover edge cases" is obscure and would be confusing to read for anybody interactively inspecting the functions in REPL or autogenerated docs. Besides, it doesn't actually escape the version, it just escapes the beginning of a possible RST substitution conditionally. In fact, there is no guarantee there would be any mentions of version present in that string.